task_spec/spec/acceptance/init_spec.rb

```diff
@@ -32,7 +32,17 @@ def target_platform
     hosts.first[:platform]
   end
 
+  def log_result(result)
```

Comment:
A little bit confusing when nothing happens for a successful `result`; maybe something like `log_output_errors` or something of that nature?

Reply:
Updated